third_party/antenna/pi.antenna.php

```diff
@@ -162,12 +166,29 @@ public function Antenna()
 	    	}
     	}
 
-    	// Inject YouTube rel value if required
-    	if (!is_null($youtube_rel) && (strpos($video_url, "youtube.com/") !== FALSE OR strpos($video_url, "youtu.be/") !== FALSE))
```

Comment:
I think this conditional might need to be enclosed in another set of parentheses to retain the intended function:

```
if (!is_null($youtube_rel) && ((strpos($video_url, "youtube.com/") !== FALSE OR strpos($video_url, "youtu.be/") !== FALSE)))
```

Not completely sure though, haven't look at what it does in-depth, just at how it changed from the original.